lambda/translquote.ml

```diff
@@ -0,0 +1,3144 @@
+
+  val unboxed_int64 : Location.t -> int64 -> t'
+
+  val unboxed_nativeint : Location.t -> nativeint -> t'
+end = struct
+  type s = lambda
+
+  type t' = s lazy_t
+
+  type t = s lam
+
+  let wrap = inject_force
+
+  let int loc x =
+    apply1 "Constant" "int" loc (Lconst (Const_base (Const_int x)))
+
+  let char loc x =
+    apply1 "Constant" "char" loc (Lconst (Const_base (Const_char x)))
+
+  let string loc a1 a2 =
+    apply2 "Constant" "string" loc
+      (Lconst (Const_base (Const_string (a1, loc, None))))
+      (string_option loc a2)
+
+  let float loc x =
+    apply1 "Constant" "float" loc
+      (Lconst (Const_base (Const_string (x, loc, None))))
+
+  let float32 loc x =
+    apply1 "Constant" "float32" loc
+      (Lconst (Const_base (Const_string (x, loc, None))))
+
+  let int32 loc x =
+    apply1 "Constant" "int32" loc (Lconst (Const_base (Const_int32 x)))
+
+  let int64 loc x =
+    apply1 "Constant" "int64" loc (Lconst (Const_base (Const_int64 x)))
+
+  let nativeint loc x =
+    apply1 "Constant" "nativeint" loc (Lconst (Const_base (Const_nativeint x)))
+
+  let unboxed_float loc x =
+    apply1 "Constant" "unboxed_float" loc
+      (Lconst (Const_base (Const_string (x, loc, None))))
+
+  let unboxed_float32 loc x =
+    apply1 "Constant" "unboxed_float32" loc
+      (Lconst (Const_base (Const_string (x, loc, None))))
+
+  let unboxed_int32 loc x =
+    apply1 "Constant" "unboxed_int32" loc
+      (Lconst (Const_base (Const_unboxed_int32 x)))
+
+  let unboxed_int64 loc x =
+    apply1 "Constant" "unboxed_int64" loc
+      (Lconst (Const_base (Const_unboxed_int64 x)))
+
+  let unboxed_nativeint loc x =
+    apply1 "Constant" "unboxed_nativeint" loc
+      (Lconst (Const_base (Const_unboxed_nativeint x)))
+end
+
+module Exp_attribute : sig
+  type s
+
+  type t' = s lazy_t
+
+  type t = s lam
+
+  val wrap : t' -> t
+
+  val inline : t'
+
+  val inlined : t'
+
+  val specialise : t'
+
+  val specialised : t'
+
+  val unrolled : t'
+
+  val nontail : t'
+
+  val tail : t'
+
+  val poll : t'
+
+  val loop : t'
+
+  val tail_mod_cons : t'
+
+  val quotation : t'
+end = struct
+  type s = lambda
+
+  type t' = s lazy_t
+
+  type t = s lam
+
+  let wrap = inject_force
+
+  let inline = use "Exp_attribute" "inline"
+
+  let inlined = use "Exp_attribute" "inlined"
+
+  let specialise = use "Exp_attribute" "specialise"
+
+  let specialised = use "Exp_attribute" "specialised"
+
+  let unrolled = use "Exp_attribute" "unrolled"
+
+  let nontail = use "Exp_attribute" "nontail"
+
+  let tail = use "Exp_attribute" "tail"
+
+  let poll = use "Exp_attribute" "poll"
+
+  let loop = use "Exp_attribute" "loop"
+
+  let tail_mod_cons = use "Exp_attribute" "tail_mod_cons"
+
+  let quotation = use "Exp_attribute" "quotation"
+end
+
+module Identifier : sig
+  module Module : sig
+    type s
+
+    type t' = s lazy_t
+
+    type t = s lam
+
+    val wrap : t' -> t
+
+    val compilation_unit : Location.t -> Global_module.Name.t -> t'
```

Comment:
It feels notable that this function is called `compilation_unit` but doesn't take a `Compilation_unit.t`